modules/scanning-images.adoc

```diff
@@ -10,17 +10,16 @@ Central submits image scanning requests to Scanner.
 Upon receiving these requests, Scanner pulls the image layers from the relevant registry, checks the images, and identifies installed packages in each layer.
 Then it compares the identified packages and programming language-specific dependencies with the vulnerability lists and sends information back to Central.
 
-You can also integrate {product-title} with another vulnerability scanner.
-//Add link to integration topics
-
-Scanner identifies the vulnerabilities in the:
+Scanner identifies the vulnerabilities in the following areas:
 
 * base image operating system
 * packages that are installed by the package managers
 * programming language specific dependencies
 * programming runtimes and frameworks
```

Comment:
Scanner V4 does not do this by the way

Reply:
I think this section still needs work, but for now, I've made it clear that this description is for the StackRox Scanner.